frontend/packages/ceph-storage-plugin/src/components/dashboard-page/storage-dashboard/capacity-breakdown/capacity-breakdown-card.scss

```diff
@@ -0,0 +1,26 @@
+.stack-chart-bottom {
+    height: 100px;
+    width: 100%;
+    padding: 0;
+    margin-top: -30px;
+}
+
+.stack-chart-legend-bottom {
+    height: 100px;
+    width: 100%;
+    padding: 0;
+    margin-top: -10px;
+}
+
+.co-capacity-breakdown-card__body {
+    padding-top: 24px !important;
+}
+
+.co-capacity-breakdown-card__header{
+    display: flex;
+}
+
+.ceph-capacity-breakdown-monitoring-header-link{
+    padding: 0.3rem;
+    margin-right: 1rem;
```

Comment:
@a2batic two different units have been used `rem` and `px`. Any reasons for that?